share/examples/bitcoin.conf

```diff
@@ -63,7 +63,12 @@
 # Port on which to listen for connections (default: 8333, testnet: 18333, signet: 38333, regtest: 18444)
 #port=
 
-# Maximum number of inbound+outbound connections.
+# Maximum number of inbound + outbound connections (default: 125). This option
+# applies only if inbound connections are enabled, otherwise the number of connections
```

Comment:
```suggestion
# applies only if inbound connections are enabled. Otherwise, the number of connections
```
or 

```suggestion
# applies only if inbound connections are enabled; otherwise, the number of connections
```

Reply:
thanks, semi-colon + comma seems good